source/includes/php/useElement.md

```diff
@@ -0,0 +1,302 @@
+var_dump($product->getApiAttributes()); // Prints 'label' and 'brand' with their values
+```
+
+To get all the attributes of an Element including the nested Elements, you can use `$element->getAllAttributes()`.
+
+To get only the attributes an Element will use for an API call, you can use `$element->getApiAttributes()`.
+
+> Get all keys an Element accepts
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+$product = new \Shoprunback\Elements\Product();
+
+var_dump($product->getApiAttributesKeys()); // Prints all attributes a Product will accept before doing an API call
+```
+
+You can list all the attributes an Element accepts when making an API call with `$product->getApiAttributesKeys()`.
+
+## Use the elements with my objects
+
+> Separate my own classes and the library's classes
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+use \Shoprunback\Elements\Product as libProduct;
+
+class myProduct
+{
+  public function __construct()
+  {
+    $this->label = 'My label';
+    $this->reference = 'my-label';
+    $this->weight_in_grams = 1000;
+  }
+
+  public function generateLibProduct()
+  {
+    $product = new libProduct();
+
+    $product->label = $this->label;
+    $product->reference = $this->reference;
+    $product->weight_in_grams = $this->weight_in_grams;
+
+    //[...]
+    // Add all params to $product
+    //[...]
+
+    return $product;
+  }
+
+  public function save()
+  {
+    // [...]
+
+    $libProduct = $this->generateLibProduct();
+    $libProduct->save();
+
+    // [...]
+  }
+}
+
+$myProduct = new myProduct();
+$myProduct->save();
+```
+
+> Extend the library's classes
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+use \Shoprunback\Elements\Product as libProduct;
+
+class myProduct extends libProduct
+{
+  public function __construct($params)
+  {
+    $this->label = $params['label'];
+    $this->reference = $params['reference'];
+    $this->weight_in_grams = $params['weight_in_grams'];
+
+    //[...]
+    // Add all params to $product
+    //[...]
+
+    if ($params['id']) {
+      parent::__construct($params['id']);
+    } else {
+      parent::__construct();
+    }
+  }
+
+  // Be careful not to overwrite the functions of the parent class!
+  public function saveProduct()
+  {
+    // Code...
+
+    $this->save(); // save() is a function of the parent class, don't overwrite it!
+
+    // Code...
+  }
+}
+
+$params = [
+  'label' => 'My label',
+  'reference' => 'my-label',
+  'weight_in_grams' => 1000,
+];
+
+$myProduct = new myProduct($params);
+$myProduct->saveProduct();
+```
+
+You can **either use the library independantly of your classes or extend the library's classes** ->
+
+**If you use inheritance**, be **careful not to overwrite the parent functions**. Instead, **use functions including the parent function**.
+
+## Check changed fields
+
+> Check if a Product is a new one or not
```

Comment:
Check if an Element is a new one or not

People could be confused and think it's only for the product